src/OSPSuite.R/Bootstrap/ApplicationStartup.cs

```diff
@@ -44,7 +44,9 @@ private IContainer performInitialization(ApiConfig apiConfig, Action<IContainer>
 
          using (container.OptimizeDependencyResolution())
          {
-            if (registerAction != null)
+            if (registerAction == null)
```

Comment:
this looks weird to me without explanation. Why do we have this weird "if" to register the group repository. I can't tell